IdentityCore/src/broker_operation/response/MSIDBrokerOperationResponse.h

```diff
@@ -44,7 +44,13 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, nullable) NSDictionary *httpHeaders;
 @property (nonatomic) NSString *httpVersion;
 
+@property (nonatomic, nullable) NSHTTPURLResponse *httpResponse;
+@property (nonatomic, nullable) NSData *httpBody;
+@property (nonatomic, nullable) NSError *httpError;
+@property (nonatomic) BOOL isBrowserResponse;
```

Comment:
Looks like this deserves a separate class from MSIDBrokerOperationResponse. 
Perhaps separate:
MSIDBrokerOperationResponse -> MSIDBrokerNativeAppOperationResponse
For browser -> MSIDBrokerBrowserResponse